PlotsBase/src/utils.jl

```diff
@@ -104,13 +104,13 @@ function _update_series_attributes!(plotattributes::AKW, plt::Plot, sp::Subplot)
 
     # update markerstrokecolor
     plotattributes[:markerstrokecolor] = if plotattributes[:markerstrokecolor] ≡ :match
-        plot_color(sp[:foreground_color_subplot])
+        get_series_color(plotattributes[:seriescolor], sp, plotIndex, stype)
```

Comment:
This causes the markerstroke to always be the same as the markercolor. I don't think we want this. E.g.
```julia
using Plots
scatter(rand(10))
```
now looks like this
![image](https://github.com/user-attachments/assets/822b55a6-cdd2-4220-9488-3f9efff683f1)